argoverse/data_loading/argoverse_stereo_loader.py

```diff
@@ -0,0 +1,438 @@
+# <Copyright 2021, Argo AI, LLC. Released under the MIT license.>
+
+import glob
+import logging
+import os
+from pathlib import Path
+from typing import Dict, Iterator, List, Optional, Union
+
+import cv2
+import numpy as np
+
+from argoverse.utils.calibration import Calibration, load_stereo_calib
+from argoverse.utils.camera_stats import RECTIFIED_STEREO_CAMERA_LIST
+
+logger = logging.getLogger(__name__)
+
+STEREO_FRONT_LEFT_RECT = RECTIFIED_STEREO_CAMERA_LIST[0]
+STEREO_FRONT_RIGHT_RECT = RECTIFIED_STEREO_CAMERA_LIST[1]
+
+
+class ArgoverseStereoLoader:
```

Comment:
i do some code repeating here vs the other ArgoverseTrackingLoader. Would it make sense to have both inherit from a parent, and then differ as children?

Which functionality is shared vs. different?

Reply:
Agreed, definitely a meta data loader would be helpful here. I'd have to carefully think about it to not break things in the forecast and tracking loaders though. The problem is that we decided to have a different data structure for the stereo dataset. We have 2 folders: rectified_stereo_images_v1.1 and disparity_maps_v1.1. Also I'm using the RECTIFIED_STEREO_CAMERA_LIST. So the methods need to be a bit different from the tracking/forecasting loader.